docs/topics/http/shortcuts.txt

```diff
@@ -110,6 +110,9 @@ This example is equivalent to::
    By default issues a temporary redirect; pass ``permanent=True`` to issue a
    permanent redirect.
 
+   By passing ``preserve_method=True`` the response signals the user agent to
+   preserve the HTTP method when doing the redirection.
+
```

Comment:
How do I mark this argument as added in version 5.2?

Reply:
Do you mean `.. versionchanged:: 5.2`? Is this an example of what you're looking for?https://github.com/django/django/blob/6765b6adf924c1bc8792a4a454d5a788c1abc98e/docs/howto/error-reporting.txt#L287

Reply:
Yes, but how to use it for an added argument and not the whole function?

Reply:
I'm not too sure. I did a search for `versionchanged`, `versionadded`, `argument.*added`, but I haven't seen an example yet that adds it to the argument. There are several examples in [testing tools](https://docs.djangoproject.com/en/5.1/topics/testing/tools/#django.test.Client.head) and elsewhere that apply the version at the function level. Would it be fine as you have it at the function level? Or would you want to add a `.. note` for the argument?

Reply:
I think it will suffice if the `.. versionchanged:: 5.2` is moved to the bottom of this section.

@lorinkoz it would be ideal if you could add an example in the following section.